modules/manage/pages/kubernetes/security/authentication/k-user-controller.adoc

```diff
@@ -0,0 +1,273 @@
+= Manage Users and ACLs with the Redpanda Operator
+:description: Use the User resource to declaratively create and manage users and ACLs as part of a Redpanda deployment. Each User resource is mapped to a user in your Redpanda cluster. The user controller keeps the corresponding user in sync with the User resource.
+:page-categories: Management, Development
+:env-kubernetes: true
+
+With the Redpanda Operator, you can declaratively create and manage Redpanda users and glossterm:ACL[,access control lists (ACLs)] using xref:reference:k-crd.adoc[User custom resources] (resources) in Kubernetes. Each User resource is mapped to a user in your Redpanda cluster. The user controller, a component of the Redpanda Operator, keeps the corresponding user in sync with the User resource.
+
+== Prerequisites
+
+You must have the following:
+
+* *Kubectl*: Ensure you have the https://kubernetes.io/docs/tasks/tools/#kubectl[kubectl^] command-line tool installed and configured to communicate with your cluster.
+* *Redpanda Operator*: Ensure you have the xref:deploy:deployment-option/self-hosted/kubernetes/k-production-deployment.adoc[Redpanda Operator].
+* *Redpanda cluster with SASL enabled*: Ensure you have a Redpanda resource deployed with xref:manage:kubernetes/security/authentication/k-authentication.adoc#enable[SASL authentication enabled].
+
+== Create a user
+
+You can use the User resource to:
+
+- <<both, Create a new user and its ACLs>>
+- <<no-acl, Create a new user without any authorization (ACLs)>>
+- <<only-acl, Create only the ACLs for a user>>
+
+Each User instance is responsible for managing both the user credentials (authentication) and the user's ACLs within the Redpanda cluster. You cannot use one User resource to manage the user and another resource to manage the ACLs. Only one User instance is allowed per user in the Redpanda cluster.
+
+[[no-acl]]
+=== Create a new user without any ACLs
+
+- *Use case*: You want to create and manage user credentials (authentication) without managing ACLs. Use this option If you have a separate process to manage ACLs or if you're working in an environment where access control is handled externally.
+- *What happens when deleted*: The user is deleted, but ACLs for that user will remain in the cluster.
```

Comment:
Found this a bit confusing--isn't the whole point of this use case that you are not using ACLs for this user? So if you delete a user, why state ACLs remain in the cluster when the use case is that you are not creating any for that user? Is this in the case that someone else has assigned ACLs to this user?

Reply:
It’s more about what happens when the User resource is deleted in Kubernetes. It might not be obvious that if ACLs aren’t defined in the User resource they remain in the cluster even after the resource is deleted.